js/main.js

```diff
@@ -102,6 +144,27 @@ function handleAnchorClick(event) {
   }
 }
 
+function handleEditClick(event) {
+  if (event.target.className === 'fa-solid fa-pencil') {
+    const $listEntryID = event.target
+      .closest('li')
+      .getAttribute('data-entry-id');
+    for (const entry of data.entries) {
+      if (entry.entryID === Number($listEntryID)) {
+        $entryFormTitle.innerText = 'Edit Entry';
+        $entryTitle.value = entry.title;
+        $photoURL.value = entry.photo;
+        $entryNotes.value = entry.notes;
+        $photo.setAttribute('src', entry.photo);
+        $photo.setAttribute('alt', 'image from photoURL');
+        data.editing = entry;
+        viewSwap();
```

Comment:
Should be passing in a string of what view you want to switch to. In this case, it should be `entry-form`